src/Pelagos/Util/UrlValidation.php

```diff
@@ -28,7 +28,8 @@ public function validateUrl($url)
 
         //check to see if it worked.
         if (200 !== $httpCode) {
-            throw new \Exception("url failed with:$httpCode($output)", $httpCode);
+
+            return "Could not get URL, returned HTTP code:$httpCode";
```

Comment:
I don't think many people know about http code. In my opinion, we can change this to something like "URL is not available (httpCode)".

Reply:
We'll run it by William and Rosalie about the error message